docs/guides/auth.md

```diff
@@ -175,26 +175,41 @@ var scope = 'https://www.googleapis.com/auth/grpc-testing';
 
 ```csharp
 // Base case - No encryption/authentication
```

Comment:
Good idea, I've made the splitting. Still not sure if it's 100% clean, but definitely looks much better than before.Ideas what more to adjust are welcome.